src/templates/dotnet/getDotnetVerifiedTemplateIds.ts

```diff
@@ -33,6 +35,6 @@ export function getDotnetVerifiedTemplateIds(version: string): RegExp[] {
     }
 
     return verifiedTemplateIds.map(id => {
-        return new RegExp(`^azure\\.function\\.csharp\\.(?:isolated\\.|)${id}\\.[0-9]+\\.x$`, 'i');
+        return new RegExp(`^azure\\.function\\.csharp\\.(?:isolated\\.|)${id}(\\.[0-9]+\\.x|)$`, 'i');
```

Comment:
After discussions with @alexweininger and @nturinski, we found that this regex specifically only looks for template `identity` names that include `.x.Version.x` alongside it to be included for the verified templates list. However, after looking over the templates.json of many extensions we found most include either `.1.x` or `.2.x` and it came to our assumption that this was the function runtimes to keep track of what is supported. However, since this seems like an extremely tedious task, it was forgotten and no longer updated for many extensions as such. 

As such, I would like to know if we could change the regex to **not** include the versioning requirement in order for GA'd SQL input/output binding can be shown on the verified lists.